cmd/rancher-desktop-guestagent/main.go

```diff
@@ -75,29 +80,60 @@ func main() {
 
 	group, ctx := errgroup.WithContext(context.Background())
 
-	if *enableDocker {
+	if *enablePriviledService && *enableIptables {
+		log.Fatal("error enabling Privileged Service mode and iptables, you can only enable one.")
+	}
+
+	if *enablePriviledService {
+		if !*enableContainerd && !*enableDocker {
+			log.Fatal("enabling Privileged Service mode requires either Docker or Containerd enabled.")
```

Comment:
```suggestion
			log.Fatal("-privilegedService mode requires either -docker or -containerd enabled.")
```
It's always clearer to use the string the user needs to actually supply in the error message (I think).